src/hooks/auth.ts

```diff
@@ -15,6 +15,12 @@ export function useLogin() {
     mutationFn: (credentials: LoginCredentials) => authApi.login(credentials),
     onSuccess: (data) => {
       queryClient.setQueryData(["auth-session"], data);
+
+      // Set admin cookie if user is admin
+      if (data.user.isAdmin) {
+        document.cookie = "kthais_admin=true; path=/; samesite=strict";
+      }
```

Comment:
I don't think we should set a an extra cookie for the user. We can use the cookies from api.kthais.com and fetch to that endpoint to verify that the user is still authenticated.